x-pack/plugins/security_solution/public/management/components/endpoint_policy_link.tsx

```diff
@@ -7,11 +7,11 @@
 
 import React, { memo, useMemo } from 'react';
 import { EuiLink, EuiLinkAnchorProps } from '@elastic/eui';
-import { useEndpointSelector } from '../hooks';
-import { nonExistingPolicies } from '../../store/selectors';
-import { getPolicyDetailPath } from '../../../../common/routing';
-import { useNavigateByRouterEventHandler } from '../../../../../common/hooks/endpoint/use_navigate_by_router_event_handler';
-import { useAppUrl } from '../../../../../common/lib/kibana/hooks';
+import { nonExistingPolicies } from '../pages/endpoint_hosts/store/selectors';
```

Comment:
It looks like when this component doesn't return back nonExistingPolicies, like right now when we are calling it from the policy list, the returned component is just a normal EuiLink. Since nonExistingPolicies is to catch the case where the metadata documents do not have the latest policy information, and we are not calling the metadata docs in the policy list ( we are calling fleet directly ), we don't really need this check for the policy list.